manifests/init.pp

```diff
@@ -115,6 +230,33 @@
     content => template('colorprompt/colorprompt.erb'),
   }
 
+  if $git_prompt {
+    unless defined(Package[$bash_completion_package]) {
+      package { $bash_completion_package:
+        ensure => present,
+      }
+    }
+    unless defined(Package[$git_package]) {
+      package { $git_package:
+        ensure => present,
+      }
+    }
+  }
+  file { "${::colorprompt::params::bash_completion_dir}/git-prompt":
```

Comment:
This file type should be within the "if $git_prompt" statement. Running Puppet with $git_prompt set to false fails with error "Error: Could not find dependency Package[bash-completion] for File[/etc/bash_completion.d/git-prompt] at /etc/puppet/modules/colorprompt/manifests/init.pp:258"